Framework/Algorithms/src/Stitch1DMany.cpp

```diff
@@ -168,18 +177,14 @@ std::map<std::string, std::string> Stitch1DMany::validateCommonInputs() {
     errors["InputWorkspaces"] = "At least 2 input workspaces required.";
 
   // Check that all the workspaces are of the same type
-  if (!m_inputWSMatrix.empty() && !m_inputWSMatrix[0].empty()) {
```

Comment:
Actually I was wrong, its still possible for it to be empty. Placing the check back in now, though I will implement it differently from before.